### DISCUSSION TITLE
Make Animation Player tracks searchable

### DISCUSSION THREAD

**Discussion by Umckgil** [godotengine/godot-proposals#7580] [Editor]
### Describe the project you are working on

Mobile games

### Describe the problem or limitation you are having in your project

I use 'find in file' quite often to remember where certain functions were called, or checking through all cases for troubleshooting etc. 

Sometimes I miss stuff because a certain function was called from an animation player. This is especially true if the code is not very organized during initial stages of development.

Please let me know if this funtion is already possible to manage in another way.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

in 'find in files' list also words included in animation player . 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

maybe just add a section in 'find in files' and an option to enable/disable it

### If this enhancement will not be used often, can it be worked around with a few lines of script?

often

### Is there a reason why this should be core and not an add-on in the asset library?

no

**Comment by KoBeWi** [comment]
You can add `tscn` to `editor/script/search_in_file_extensions` project setting and it will search inside scenes too.

**Reply by Umckgil** [reply]
hi thanks this tip works for my purpose. it at least shows that there is something in the tscn text file with that word.

edit: just noticed not that it already links to the related animation player as well. so closing the discussion
